api-doc/index.md

```diff
@@ -0,0 +1,192 @@
+# gfoidl.Base64
```

Comment:
It's quite the same as the ReadMe, so maybe it's possible to link it?
On the other side, iif the contents should diverge by different means, so it's better to keep this file as is.